pkg/services/rpcsrv/client_test.go

```diff
@@ -1723,7 +1723,7 @@ func TestClient_IteratorSessions(t *testing.T) {
 			require.True(t, ok)
 
 			ok, err = c.TerminateSession(sID)
-			require.NoError(t, err)
+			require.Error(t, err)
```

Comment:
I consider we should do the following wrt this `(c *Client) TerminateSession` behaviour change:
1. Add `if errors.Is(err, neorpc.ErrUnknownSession) {return false, nil}` branch of code to preserve the old behaviour for a couple of releases, because it's a user-facing API that can be used by someone (till 0.105.0 release is probably sufficient).
2. Add a comment about this behaviour to `(c *Client) TerminateSession`.
3. Add a note to the end of `# Deprecated functionality section of the `ROADMAP.md` file with scheduled removal of old behaviour support.

@roman-khimov, or maybe we don't need this at all and an instant change is more preferable?

Reply:
If you hide this error in the client for now, the code isn't likely to be updated and it'll break when you're to remove compatibility code. It'll also make it impossible to handle the error for the code that wants to do that. So we'll need a warning in release notes about this particular RPC, but we should leave the client as is. In normal use it's not a noticeable change.

Reply:
Sorry, @roman-khimov @AnnaShaleva I didn't catch should I do it here or in the separate branch/issue?